packages/atomic/src/components/common/template-system/cell-desktop.pcss

```diff
@@ -4,11 +4,11 @@
   grid-template-areas:
     'badges'
     'visual'
-    'children'
     'title'
     'title-metadata'
     'emphasized'
     'excerpt'
+    'children'
```

Comment:
What does this do ?

Reply:
puts the variants at the bottom, preventing the layout shifts when some products dont have variants. This is what is shown in the before/after images

Reply:
@alexprudhomme actually gonna move these two changes in a separate PR I think